server/odc-service/src/main/java/com/oceanbase/odc/service/connection/database/DatabaseService.java

```diff
@@ -335,7 +335,7 @@ public List<ConnectionConfig> statsConnectionConfig() {
         return databases.stream().filter(database -> Objects.nonNull(database.getDataSource()))
                 .map(Database::getDataSource)
                 .collect(Collectors.collectingAndThen(
-                        Collectors.toCollection(() -> new TreeSet<>(Comparator.comparing(ConnectionConfig::getId))),
+                        Collectors.toCollection(() -> new TreeSet<>(Comparator.comparing(ConnectionConfig::getName))),
```

Comment:
This is for stats, so it's unnecessary to order here.

Reply:
used for sort data sources by name in the sql console